docs/robots/husky/tutorials_husky.mdx

```diff
@@ -4,7 +4,1535 @@ sidebar_position: 4
+
+If your Husky does not have the ``husky_tests`` ROS package installed already, you can manually install it by opening terminal and running:
+
+```bash
+sudo apt-get install ros-noetic-husky-tests
+```
+
+### Running ROS Tests
+
+The `ros_tests` script exposes a set of interactive tests to verify the functionality of core features.
+These tests run at the ROS-level via ROS topics, and serve as a useful robot-level diagnostic tool for
+identifying the root cause of problems, or at the very least, narrowing down on where the root cause(s) may be.
+
+To run `ros_tests` on a Husky robot, open terminal and run:
+
+```bash
+rosrun husky_tests ros_tests
+```
+
+Upon running `ros_tests`, a list of available tests will be shown in a menu. From the menu, you can choose
+individual tests to run, or simply choose the option to automatically run all the tests.
+
+The details of each test are shown below.
+
+- **E-Stop Test**
+
+    Checks that the robot's E-Stop is working properly. 
+
+    This test subscribes to the `/status` ROS topic and checks that when the E-Stop is manually engaged by
+    the user, the E-Stop state is correctly reported on the `/status` ROS topic.
+
+- **ADC Test**
+
+    Checks that the robot's voltage and current values across its internal hardware components are within
+    expected tolerances.
+
+    This test subscribes to the `/status` ROS topic and checks that the voltage and current values across
+    the internal hardware are within expected tolerances.
+
+- **Rotate Test**  (Optional test; only applies to Husky's with an IMU)
+
+    Rotates the robot counter clockwise 2 full revolutions and checks that the motors, IMU, and EKF odometry are working properly.
+
+    This test:
+
+    - Subscribes to the `/imu/data` ROS topic to receive angular velocity measurements from the IMU's Gyroscope.
+      These measurements are converted into angular displacement estimations, and the robot will rotate until 2
+      full revolutions are estimated.
+    - Subscribes to the `/odometry/filtered` ROS topic to receive angular velocity estimations from the EKF
+      odometry. These measurements are converted into angular displacement estimations, and are output as comparison
+      to the angular displacement estimations from the IMU's Gyroscope.
+    - Publishes to the `/cmd_vel` ROS topic to send drive commands to rotate the robot.
+    - The user will be asked to verify that the robot rotates 2 full revolutions.
+
+    :::note
+    The **Rotate Test** rotates the robot using the IMU's Gyroscope data, which inherently will not be 100% accurate.
+    Therefore, some undershoot/overshoot is to be expected.
+    :::
+
+- **Drive Test**
+
+    Drives the robot forward 1 meter and checks that the motors, encoders, and encoder-fused odometry are working properly.
```

Comment:
Note, though this does not require an update:

- I have been spelling meter as metre.
- We could add common spellings and preferred words to our README:
  - meter vs. metre
  - E-Stop vs. Motion-Stop
  - 8mm vs. 8mm
  - 2x4 vs 2 X 4
  - 2x vs 2X

Reply:
I've changed to 'metre' here and in the Dingo tutorials. Agreed it would be good to have a set of conventions somewhere.